04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,25 @@
-#write your code here
+# Rule 1: If a word begins with a vowel sound, add an 
+#"ay" sound to the end of the word.
+#
+# Rule 2: If a word begins with a consonant sound, move 
+#it to the end of the word, and then add an "ay" sound 
+#to the end of the word.
+
+
+def translate(str)
+    str.gsub!(/\w+/) { |word| translate_word(word)}
+end
+
+def translate_word(phrase)
+    phrase.split(" ").map do |word|
```

Comment:
I dont believe you need to `split` here, because `gsub!` will run the block for each occurance of a word